lab1/main_test.js

```diff
@@ -2,22 +2,62 @@ const test = require('node:test');
 const assert = require('assert');
 const { MyClass, Student } = require('./main');
 
+// const myClass = new MyClass();
+// const names = ['John', 'Jane', 'Doe', 'Smith'];
+// names.forEach(name => {
+//     const student = new Student();
+//     student.setName(name);
+//     const newStudentId = myClass.addStudent(student);
+//     const newStudentName = myClass.getStudentById(newStudentId).getName();
+//     console.log('[+] Added student with id: %d, name: %s', newStudentId, newStudentName);
+// });
+
```

Comment:
You should not leave a bunch of comment code in your commit.
I will merge the PR first, but it's better to remove them next time.